[2023-12-01 16:07:03,588] [ERROR] [_internal] 152.74.52.235 - - [01/Dec/2023 16:07:03] code 400, message Bad request version ('ÊÊ\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x01\x93\x9a\x9a\x00\x00\x00')
[2023-12-01 16:07:03,591] [ERROR] [_internal] 152.74.52.235 - - [01/Dec/2023 16:07:03] code 400, message Bad request version ('t\x88ÝJ5')
[2023-12-06 14:54:46,744] [ERROR] [main] Ocurrió un error al agregar la carpeta del ejercicio: integer division or modulo by zero
[2023-12-06 14:54:46,771] [ERROR] [main] Ocurrió un error al agregar la carpeta de la serie: integer division or modulo by zero
[2023-12-06 14:54:57,388] [ERROR] [main] Ocurrió un error al agregar la carpeta del ejercicio: integer division or modulo by zero
[2023-12-06 14:54:57,395] [ERROR] [main] Ocurrió un error al agregar la carpeta de la serie: integer division or modulo by zero[2023-12-07 13:42:00,204] [ERROR] [__init__] Error: Path doesn't exist: '/home/ivonne/Documentos/MemoriaTituloIvonne/migrations'.  Please use the 'init' command to create a new scripts folder.
[2023-12-07 13:43:00,134] [ERROR] [__init__] Error: Path doesn't exist: '/home/ivonne/Documentos/MemoriaTituloIvonne/migrations'.  Please use the 'init' command to create a new scripts folder.
[2023-12-07 14:28:05,995] [ERROR] [main] Ocurrió un error al agregar el ejercicio: (sqlite3.IntegrityError) UNIQUE constraint failed: serie_asignada.id_serie, serie_asignada.id_grupo
[SQL: INSERT INTO serie_asignada (id_serie, id_grupo) VALUES (?, ?)]
[parameters: ('1', '1')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
[2023-12-11 23:30:26,274] [ERROR] [main] Ocurrió un error al agregar el ejercicio: (sqlite3.IntegrityError) UNIQUE constraint failed: ejercicio.nombre
[SQL: INSERT INTO ejercicio (nombre, path_ejercicio, enunciado, id_serie) VALUES (?, ?, ?, ?)]
[parameters: ('Tienda mágica', '', '', '1')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
[2023-12-14 14:31:56,583] [ERROR] [main] Ocurrió un error al agregar el ejercicio: (sqlite3.IntegrityError) UNIQUE constraint failed: serie_asignada.id_serie, serie_asignada.id_grupo
[SQL: INSERT INTO serie_asignada (id_serie, id_grupo) VALUES (?, ?)]
[parameters: ('1', '1')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
[2023-12-14 17:47:36,821] [ERROR] [main] Ocurrió un error al agregar el ejercicio: (sqlite3.IntegrityError) UNIQUE constraint failed: serie_asignada.id_serie, serie_asignada.id_grupo
[SQL: INSERT INTO serie_asignada (id_serie, id_grupo) VALUES (?, ?)]
[parameters: ('1', '1')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
[2023-12-21 17:34:03,713] [ERROR] [main] Ocurrió un error al agregar la carpeta del ejercicio: Cannot insert None into a tag.
[2023-12-21 17:34:03,713] [ERROR] [main] Ocurrió un error al agregar la carpeta de la serie: local variable 'resultadoTest' referenced before assignment
[2023-12-21 17:34:16,941] [ERROR] [main] Ocurrió un error al agregar la carpeta del ejercicio: Cannot insert None into a tag.
[2023-12-21 17:34:16,941] [ERROR] [main] Ocurrió un error al agregar la carpeta de la serie: local variable 'resultadoTest' referenced before assignment
[2023-12-22 09:37:02,124] [ERROR] [main] Ocurrió un error al agregar el ejercicio: (sqlite3.IntegrityError) UNIQUE constraint failed: ejercicio.nombre
[SQL: INSERT INTO ejercicio (nombre, path_ejercicio, enunciado, id_serie) VALUES (?, ?, ?, ?)]
[parameters: ('Tienda mágica p2', '', '', '4')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
